test/GPv2Settlement.test.ts

```diff
@@ -31,11 +37,27 @@ describe("GPv2Settlement", () => {
       const GPv2Settlement = await ethers.getContractFactory(
         "GPv2SettlementTestInterface",
       );
-      const settlement2 = await GPv2Settlement.deploy();
+      const settlement2 = await GPv2Settlement.deploy(controller.address);
 
       expect(await settlement.domainSeparatorTest()).to.not.equal(
         await settlement2.domainSeparatorTest(),
       );
     });
   });
+
+  describe("settle", () => {
+    it("rejects transactions from non-solvers", async () => {
+      await expect(settlement.settle([], [], 0, [], [], [])).to.be.revertedWith(
+        "GPv2: not a solver",
+      );
+    });
+    it("accepts transactions from solvers", async () => {
```

Comment:
ubernit: please add new-lines between test cases

Reply:
```suggestion

    it("accepts transactions from solvers", async () => {
```

Reply:
I'm trying! but sometimes it just slips. Will eventually make it a habit.

Reply:
I understand Ben's struggle. :smile: